src/processors/modulation/Rotary.cpp

```diff
@@ -154,12 +164,16 @@ void Rotary::processAudio (AudioBuffer<float>& buffer)
 
     if (inputsConnected.contains (AudioInput))
     {
-        const auto& audioInputBuffer = getInputBuffer (AudioInput);
-        const auto numChannels = audioInputBuffer.getNumChannels();
-        audioOutBuffer.makeCopyOf (audioInputBuffer, true);
+        const auto stereoMode = stereoParam->get();
+        const auto& audioInBuffer = getInputBuffer (AudioInput);
+        const auto numInChannels = audioInBuffer.getNumChannels();
+        const auto numOutChannels = stereoMode ? 2 : numInChannels;
+
+        audioOutBuffer.setSize (numOutChannels, numSamples, false, false, true);
 
-        for (int ch = 0; ch < numChannels; ++ch)
+        for (int ch = 0; ch < numOutChannels; ++ch)
         {
+            audioOutBuffer.copyFrom (ch, 0, audioInBuffer, ch % numInChannels, 0, numSamples);
```

Comment:
So if the input signal is stereo then the stereo parameter will have no effect? I think that makes sense here.